tests/integration/cli/test_cli.py

```diff
@@ -222,6 +231,17 @@ def test_otx_explain_e2e(
     assert (tmp_path_explain / "outputs").exists()
     assert (tmp_path_explain / "outputs" / "saliency_map.tiff").exists()
 
+    reference_sal_vals = {
```

Comment:
Theses two models are enough to test?

Reply:
It is not, but there is some issues with reproducibility. I will extend the scope in the next PRs.